### PR TITLE
yoav - DO THIS SECOND!!! Auto Find Ks (2 files)

### PR DESCRIPTION
**Pull request by YHerm**

### CODE REVIEW

File: src/main/java/frc/utils/calibration/autocalibration/findks/FindKs.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        lastTimeSeconds = TIMER.get();
+    }
+
+    @Override
+    public void execute() {
+        if (TIMER.get() - lastTimeSeconds >= FindKsConstants.TIME_BETWEEN_VOLTAGE_RAMPS_SECONDS) {
+            lastVoltage += FindKsConstants.VOLTAGE_STEP;
+            setVoltageControl.accept(lastVoltage + kG);
+            lastTimeSeconds = TIMER.get();
+        }
```
[the remark this answers is not in the document]
Comment (YHerm) [reply src/main/java/frc/utils/calibration/autocalibration/findks/FindKs.java:49]: how it handeled?

[the remark this answers is not in the document]
Comment (YHerm) [reply src/main/java/frc/utils/calibration/autocalibration/findks/FindKs.java:49]: you cant, in rio it reset the current time each rio cycle, and here it is for a specific time difference.
you can do a class that warps timer and add some stuff, like get specific time difference, but i dont see reason now

[the remark this answers is not in the document]
Comment (YoniKiriaty) [reply src/main/java/frc/utils/calibration/autocalibration/findks/FindKs.java:49]: i know, but in CRESCENDO we had a `roborioCycleUtils` class or something which tracked the current roborio time and last roborio time, and you can do a `getRoborioCycleTime` that is just `Roborio.getCycle() - lastTime`

[the remark this answers is not in the document]
Comment (YHerm) [reply src/main/java/frc/utils/calibration/autocalibration/findks/FindKs.java:49]: No the time it is returning is the cycle time, between 20 to 50 milliseconds

**Pull request closed without merging**